Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -86,11 +86,24 @@ ProjectPamphletViewModelOutputs {
       .take(first: 1)
       .map { $0 ?? $1 }
 
-    Signal.combineLatest(project, refTag, cookieRefTag)
-      .takeWhen(self.viewDidAppearAnimated.signal)
+    // Try getting array of live streams from project, but if we can't after 5 seconds let's just emit `nil`
+    let projectLiveStreams: Signal<[Project.LiveStream]?, NoError> = project
+      .map { $0.liveStreams }
+      .skipNil()
+      .timeout(after: 5, raising: SomeError(), on: AppEnvironment.current.scheduler)
+      .materialize()
+      .map { $0.value }
+      .take(first: 1)
```

Comment:
^^^ this is the stuff justin was talking about in the description of this pr.

we want to track the `Viewed Project Page` event with a property that determines if there were any live streams available or not. unfortunately, we typically don't have that data right away... we have to wait until we get fresh data from the api. but, we also don't want to miss out on an event if that data never comes back (api error, network flakiness, etc...), so if after 5 seconds we get nothing we just default to `nil` and interpret that as not having any live stream data.